app/src/main/java/org/resolvetosavelives/red/patient/PatientSearchResult.kt

```diff
@@ -69,8 +69,14 @@ data class PatientSearchResult(
     @Query("$mainQuery WHERE P.fullname LIKE '%' || :query || '%' OR PP.number LIKE '%' || :query || '%'")
     fun search(query: String): Flowable<List<PatientSearchResult>>
 
-    @Query(mainQuery)
-    fun allRecords(): Flowable<List<PatientSearchResult>>
+    @Query("""$mainQuery
+      WHERE (P.fullname LIKE '%' || :query || '%' OR PP.number LIKE '%' || :query || '%')
+      AND ((P.dateOfBirth BETWEEN :dobUpperBound AND :dobLowerBound) OR (P.age_computedDateOfBirth BETWEEN :dobUpperBound AND :dobLowerBound))
+      """)
+    fun search(query: String, dobUpperBound: String, dobLowerBound: String): Flowable<List<PatientSearchResult>>
+
+    @Query("$mainQuery ORDER BY P.updatedAt LIMIT 100")
+    fun recentlyUpdated100Records(): Flowable<List<PatientSearchResult>>
```

Comment:
Hey… shouldn't this be ordered by latest first (DESC)?

Reply:
You're right, fixed.